app/views/charge.njk

```diff
@@ -6,8 +6,13 @@
   {{ __("cardDetails.title") }}
 {% endblock %}
 
+{% set bodyClasses = "google-pay-unavailable" %}
```

Comment:
Do we actually need both these `-available` and `unavailable` classes? Can’t we just have the latter and use its absence to denote unavailability?

Reply:
It makes the CSS easier and less overiding this way